src/SDKs/OperationalInsights/Management.OperationalInsights/Microsoft.Azure.Management.OperationalInsights.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.Management.OperationalInsights</PackageId>
     <Description>Provides Microsoft Azure Operational Insights management operations</Description>
     <AssemblyName>Microsoft.Azure.Management.OperationalInsights</AssemblyName>
-    <VersionPrefix>0.18.0-preview</VersionPrefix>
+    <VersionPrefix>0.19.0-preview</VersionPrefix>
     <PackageTags>Microsoft Azure OperationalInsights;OperationalInsights</PackageTags>    
   </PropertyGroup>
```

Comment:
@sw47 add package release notes as below
Add < PackageReleaseNotes > and add Release notes.
I would recommend to use < ![CDATA and then format your Release notes

https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj#L10